md/deploy-bundle.md

```diff
@@ -80,6 +80,8 @@ use the [jdbc driver](database-configuration.md#proprietary_jdbc_drivers) provid
 
 ## Wildfly installation
 
+**WARNING:** Starting from Bonita 7.9.0 the Wildfly bundle is deprecated. It will be removed in a future release. Check the [Release Note](release-notes.md) for more details.
```

Comment:
please use the `::: warning` notation to emphasis the text
```suggestion
::: warning
**WARNING:** Starting from Bonita 7.9.0 the Wildfly bundle is deprecated. It will be removed in a future release. Check the [Release Note](release-notes.md) for more details.
:::
```